README.md

```diff
@@ -2,109 +2,122 @@
 
 [![License][badge-license]][badge-url-license]
 [![Build_status][badge-build-status]][badge-url-build-status]
-[![Coverage Status][badge-coverage]][badge-url-coverage]
+[![Coverage Status][coverage-badge-svg]][badge-url-coverage]
 
 ## Description
 
-**Pubgrade** (**Pub**lish-Up**grade**) is a decoupled, publish-subscribe-based
-continuous integration (CI) and continuous delivery (CD) service that allows
-developers to notify deployments of available updates. These can can then
-autonomously decide what to do with them. Pubgrade has an optional link with
-the [ELIXIR Cloud Service Registry][elixir-cloud-registry].
+**Pubgrade** (**Pub**lish-Up**grade**) is a decoupled, publish-subscribe-based continuous integration (CI) and continuous delivery (CD) microservice that allows developers to notify deployments of available updates, which can then autonomously decide what to do with them. Pubgrade has an optional link with [ELIXIR Cloud Service Registry][elixir-cloud-registry].
 
 Currently supported features:
 
-- Register Git repositories
-- Create and publish builds
-- Register subscriptions
-- Notify subscriptions on updated builds
+- Register git repository.
+- Create and publish builds.
+- Register subscriptions.
+- Notify subscriptions on build updates.
 
-![app-schema][anim-schema]
+![app-schema](images/app-schema-animation.gif)
 
 ## Installation
 
-1. Install and setup [Kubernetes][kubernetes-install].
+1. Install and setup [kubernetes][kubernetes-install].
 2. Install [Helm][helm-install].
-3. Modify `deployment/values.yaml`.
-4. Create a namespace, e.g., with `kubectl create namespace pubgrade`
-5. Create a deployment using Helm with `helm install pubgrade deployment/ -n
-   pubgrade`
+3. Modify deployment/values.yaml.
+4. Create a namespace. `kubectl create namespace pubgrade`
+5. Create deployment using helm. `helm install pubgrade deployment/ -n pubgrade`
 
 ## Usage
 
-Register Git repository:
+1. Register git repository.
 
 ```bash
-    curl --location --request POST 'http://{host_url}:{host_port}/repositories' \
+    curl --location --request POST 'http://<host_url>:<host_port>/repositories' \
 --header 'Content-Type: application/json' \
 --data-raw '{
-    "url": "{ repository_url }"
+    "url": "<repository_url>"
 }'
 ```
 
-Subscribe to repository (use admin user):
+2. Create a user (System administrator).
 
 ```bash
-curl --location --request POST 'http://{host_url}:{host_port}/subscriptions' \
---header 'X-User-Access-Token: { user_access_token }' \
---header 'X-User-Id: { user_identifier }' \
+curl --location --request POST 'http://<host_url>:<host_port>/users/register' \
 --header 'Content-Type: application/json' \
 --data-raw '{
-    "repository_id": "{ repository_identifier }",
-    "callback_url": "{ url }",
-    "access_token": "{ access_token }",
-    "type": "{ branch OR tag }",
-    "value": "{ branch_name OR tag_name }"
+    "name": "<name>"
```

Comment:
Why no user access token header?

Reply:
So anyone will be able to create users? even bots?

Reply:
Yes, this was I discussed earlier that anyone can register to pubgrade but won't be able to use it as account needs to verified by the administrator. Do we need any improvement here?

Reply:
We should talk about this again. What does prevent someone to create an infinite number of accounts??

Reply:
Creating an issue for that

Reply:
Ok, closing this conversation then